spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/BlockingBackPressureHandler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.listener;
+
+/**
+ * Marker interface for a blocking {@link BackPressureHandler}. This handler is used to control the flow of messages in
+ * a blocking manner. It is recommended to have at least one blocking back pressure handler in a
```

Comment:
Considering we are able to block in the Composite BPH, is this recommendation valid?

If all BPHs are non-blocking, it'd correctly enable polling until one returns 0, which would trigger the CompositeBPH to hold off the thread.

Makes sense?